services/crypto/key/controls.yaml

```diff
@@ -7,73 +7,124 @@ shared-controls:
       - CCC.C04 # Log all access and changes
       - CCC.C05 # Prevent access from untrusted entities
       - CCC.C06 # Prevent deployment in restricted regions
-      - CCC.C10 # Prevent Data Replication to Destinations Outside of Defined Trust Perimeter
+      - CCC.C10 # Prevent Data Replication to Destinations Outside Perimeter
 
 control-families:
   - name: Logging and Monitoring
-    description: 'TODO: Describe this control family'
+    description: Controls that collect, alert, and retain key-management events.
+
     controls:
       - id: CCC.KeyMgmt.C01
-        title: Alert when key versions are deleted or disabled
-        objective: |
-          Ensure that alerts are generated when key versions are scheduled for deletion or disabled.
+        title: Alert on key-version changes
+        objective: >
```

Comment:
hah, I just remembered that this exists. We should definitely start using these `>` "folding" text blocks everywhere, instead of preserving newlines with `|`.